webhooks-extension/README.md

```diff
@@ -2,235 +2,38 @@
-```
-
-`$openshift_master_default_subdomain` in this example is `mycluster.foo.com`. This gives you the following Route:
-
-```
-NAME                                    HOST/PORT                                                         PATH      SERVICES               PORT      TERMINATION          WILDCARD
-webhooks-route                          wildcard.tekton-pipelines.mycluster.foo.com                       istio-ingressgateway             http2                          Subdomain
-```
-
-Configure your ConfigMap `config-domain` in the `knative-serving` namespace: a working set up can be achieved with:
-
-```
-kubectl patch configmap config-domain --namespace knative-serving --type='json' \
-  --patch '[{"op": "add", "path": "/data/'"${openshift_master_default_subdomain}"'", "value": ""}]'
-```
-
-You can now proceed to create webhooks using the Webhooks Extension UI. Remember that your source code repository must be able to reach your cluster, or your webhooks will never be received (mentioning "Service Timeout" errors.
-
-This has been tested with the following scc (from `oc get scc`):
-
-```
-NAME               PRIV      CAPS      SELINUX     RUNASUSER          FSGROUP     SUPGROUP    PRIORITY   READONLYROOTFS   VOLUMES
-anyuid             false     []        MustRunAs   RunAsAny           RunAsAny    RunAsAny    10         false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
-hostaccess         false     []        MustRunAs   MustRunAsRange     MustRunAs   RunAsAny    <none>     false            [configMap downwardAPI emptyDir hostPath persistentVolumeClaim projected secret]
-hostmount-anyuid   false     []        MustRunAs   RunAsAny           RunAsAny    RunAsAny    <none>     false            [configMap downwardAPI emptyDir hostPath nfs persistentVolumeClaim projected secret]
-hostnetwork        false     []        MustRunAs   MustRunAsRange     MustRunAs   MustRunAs   <none>     false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
-node-exporter      false     []        RunAsAny    RunAsAny           RunAsAny    RunAsAny    <none>     false            [*]
-nonroot            false     []        MustRunAs   MustRunAsNonRoot   RunAsAny    RunAsAny    <none>     false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
-privileged         true      [*]       RunAsAny    RunAsAny           RunAsAny    RunAsAny    <none>     false            [*]
-restricted         false     []        MustRunAs   MustRunAsRange     MustRunAs   RunAsAny    <none>     false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
-```
+### Uninstall
 
+[Uninstall](./docs/Uninstall.md)
 
 ## Want to get involved?
 
 Visit the [Tekton Community](https://github.com/tektoncd/community) project for an overview of our processes.
 
-
-## For developers
+## Information For developers
```

Comment:
Capitalise the D in Developers to be consistent

Reply:
I've gone the other way and lowercased the f ..... to be consistent with "Want to get involved?"